Fix relocation handling in assembler where apparently someone overzealously
tried to get rid of some warnings.

--- cctools-855/as/ppc.c.orig	2014-04-05 00:42:22.000000000 +0200
+++ cctools-855/as/ppc.c	2015-02-20 18:40:17.000000000 +0100
@@ -1019,6 +1019,8 @@
 
 	/* if this instruction requires labels mark it for later */
 	switch(insn.reloc){
+	case NO_RELOC:
+	    break;
 	case PPC_RELOC_HI16:
 	case PPC_RELOC_LO16:
 	case PPC_RELOC_HA16:
@@ -1033,6 +1035,7 @@
 		    insn.reloc);
 	    break;
 	case PPC_RELOC_BR14:
+	case PPC_RELOC_BR14_predicted:
 	    fix_new(frag_now,
 		    thisfrag - frag_now->fr_literal,
 		    4,
